lib/sisimai/bite/json/sendgrid.rb

```diff
@@ -11,65 +11,56 @@ def description; return 'SendGrid(JSON): http://sendgrid.com/'; end
 
       # @abstract Adapt SendGrid bounce object for Sisimai::Message format
       # @param        [Hash] argvs  bounce object(JSON) retrieved from SendGrid API
-      # @return       [Hash, Nil]   Bounce data list and message/rfc822 part
-      #                             or Undef if it failed to parse or the
+      # @return       [Hash, Nil]   Bounce data list and message/rfc822 part or
```

Comment:
__[rubocop]__
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->